pkgs/servers/home-assistant/default.nix

```diff
@@ -179,10 +179,13 @@ in with py.pkgs; buildPythonApplication rec {
 
   postPatch = ''
     substituteInPlace setup.py \
+      --replace "async_timeout==3.0.1" "async_timeout" \
```

Comment:
Looks very similar to my fix: https://github.com/Mic92/nixpkgs/commit/4bc1481d6d7bc9caa726f0c2bcc3978c95646f56